proxy/proxy.go

```diff
@@ -148,8 +147,9 @@ func (proxy *Proxy) ListenAndServe() error {
 func (proxy *Proxy) weaveCIDRsFromConfig(config *docker.Config, hostConfig *docker.HostConfig) ([]string, error) {
 	if hostConfig != nil &&
 		hostConfig.NetworkMode != "" &&
+		hostConfig.NetworkMode != "default" &&
```

Comment:
@rade default was added recently as part of the `default-network-driver` support in order to make a remote network plugin such as weave as the default network driver. an empty network-mode "" is considered seen as `default` instead of assuming it as `bridge` in order to make sure the plugin can be the default.
But if there is no `default-network-driver` mentioned in the daemon, then `bridge` is used by default.
The behavior is the same in the experimental channel right after 1.7 release.